From 100e280d3e99112067f833cf36f447beaa6d31bd Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Thu, 16 Aug 2018 13:15:52 -0700
Subject: [PATCH 12/13] lineage-sdk: Update attr.xml for aapt2

*) the sdk is now using aapt2

*) aapt2 is unhappy with "lineage:..." name attribute value format
   (errors with invalid character).

*) update in a way that aapt2 is happy with

Change-Id: I96ecc14c4a6de00272a5ad4f513b62f21289ac45
---
 sdk/res/res/values/attrs.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/sdk/res/res/values/attrs.xml b/sdk/res/res/values/attrs.xml
index fe3c722..c9f1297 100644
--- a/sdk/res/res/values/attrs.xml
+++ b/sdk/res/res/values/attrs.xml
@@ -14,8 +14,8 @@
      See the License for the specific language governing permissions and
      limitations under the License.
 -->
-<resources xmlns:lineage="http://schemas.android.com/apk/res/lineageos.platform">
-    <declare-styleable name="lineage:SelfRemovingPreference">
+<resources>
+    <declare-styleable name="lineage_SelfRemovingPreference">
         <attr name="requiresPackage" format="string" />
         <attr name="requiresFeature" format="string" />
         <attr name="requiresConfig" format="string" />
@@ -28,7 +28,7 @@
     </declare-styleable>
 
     <!--Settings search providers-->
-    <declare-styleable name="lineage:Searchable">
+    <declare-styleable name="lineage_Searchable">
         <attr name="xmlRes" format="reference" />
     </declare-styleable>
 
-- 
2.17.1

